### DISCUSSION TITLE
I had to add python3-netaddr to make it work

### DISCUSSION THREAD

**Discussion by mcncyo** [techno-tim/k3s-ansible#203] [Questions]
Just to let you know.  If you get something about netaddr you need to run this command.  

sudo apt install --no-install-recommends python3-netaddr

The run reset command before you run the site.yml file

**Comment by alichaudry** [comment]
This is correct, I also had to install `netaddr` manually in my python virtual env (via conda/mamba) to get this to work. Also, I didn't have to manually install the ansible requirements in this repo as they already existed in my fresh ansible install by default.

**Comment by ildoc** [comment]
I installed ansible via pipx, so I had to add netaddr with `pipx inject ansible netaddr`
